### DISCUSSION TITLE
How to auto Update SpotX to newer Versions

### DISCUSSION THREAD

**Discussion by Prathamk911** [SpotX-Official/SpotX#371] [Question]
Can someone explain to me how to auto update SpotX to newer versions. i'm not friendly with command line. i've amazed by the Direct Installer.

**Comment by amd64fox** [comment]
SpotX doesn't auto-update at the moment
to update you need to run the SpotX installer manually again

**Reply by DogeTOM1143** [reply]
hello a question when will you guys update spotx to the newest spotify version???

**Reply by amd64fox** [reply]
The latest version now is 1.2.40.599 and spotx supports it

**Comment by jyzeng77** [comment]
Can you guys implement this feature? It's really annoying to repatch every new spotify update

**Comment by berthshoe** [comment]
Follow the new versions and click "Install_New_theme.bat" or "Install_Old_theme.bat"
:)

**Reply by berthshoe** [reply]
@jyzeng77

**Reply by jyzeng77** [reply]
Lol atp I'll just copy paste the powershell script.

**Comment by DogeTOM1143** [comment]
Yea but like i do not see the latest update ui screen tho its just the same old ui

**Comment by jyzeng77** [comment]
You have to add flags when you install it to add features/UI that isn't
toggled on by default
